fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeFunctionTest.java

```diff
@@ -75,14 +76,42 @@ public void testDateTrunc() {
 
     @Test
     public void testDateFloor() {
-        analyzeSuccess("select time_slice(th, interval 1 year) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 month) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 day) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 week) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 quarter) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 hour) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 minute) from tall");
-        analyzeSuccess("select time_slice(th, interval 1 second) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 year) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 month) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 day) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 week) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 quarter) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 hour) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 minute) from tall");
+        analyzeOnceSuccess("select time_slice(th, interval 1 second) from tall");
+
+        // Use other texts.
+        analyzeOnceSuccess("select time_slice(th, interval 1 year, fjwel) from tall");
```

Comment:
why success?

Reply:
> why success?

If with other text, this as default(START), this consistent with before.